src/Plugin.Maui.ScreenRecording/ScreenRecordingOptions.cs

```diff
@@ -21,4 +21,18 @@ public class ScreenRecordingOptions
 	/// Default value is <see langword="false"/>.
 	/// </summary>
 	public bool EnableMicrophone { get; set; }
+
+	/// <summary>
+	/// Gets or sets the notification content title.
+	/// Default value is "Screen Recording".
+	/// </summary>
+	/// <remarks>This property only has effect on Android. On other platforms no notification is shown when a screen recording is being made.</remarks>
+	public string NotificationContentTitle { get; set; }
```

Comment:
Suggestion: 
`public string NotificationContentTitle { get; set; } = string.Empty`
Then you get rid of the ugly warnings :)

Reply:
I tried, but is better to keep the null warning because if you set the property to `string.Empty`, will avoid the following validation
`NotificationContentTitle = options?.NotificationContentTitle ?? "Screen Recording";`

because Empty is not null, will always be true and if you don't set a custom text, will be empty text

I guess for this, the better option is to set the null warning to off `#nullable disable` for this class

Reply:
I tried, but is better to keep the null warning because if you set the property to string.Empty, will avoid the following validation
NotificationContentTitle = options?.NotificationContentTitle ?? "Screen Recording";

because Empty is not null, will always be true and if you don't set a custom text, will be empty text

I guess for this, the better option is to set the null warning to off #nullable disable for this class

Reply:
@m3xpl4y is well suggested, but I think the other two approaches to solve this are

using this tag a the top off the class `#nullable disable` to set nulls to off 
or
set the default value in the property instead `public string NotificationContentTitle { get; set; } = "Screen Recording";` and remove the null validation changing the code from `NotificationContentTitle = options?.NotificationContentTitle ?? "Screen Recording";` to `NotificationContentTitle = options?.NotificationContentTitle;`

I think @jfversluis could give us an opinion also.

Reply:
i did clone the rep my self and i am trying now to implement the recording for windows. 
I tell you, i don't know if this will work. Will create a PR. You can take a look